### DISCUSSION TITLE
请问该怎么实现对于某个群的所有消息，执行同一个命令

### DISCUSSION THREAD

**Discussion by Gligameshes** [nonebot/discussions#178] [问答]
比如说把一个群的消息都转发到另一个群

**Comment by yanyongyu** [comment]
使用on_message并编写一个rule限定GroupMessageEvent，只在你想要的群触发即可。然后把收到的消息send到另一个群。

**Reply by Gligameshes** [reply]
好的， 谢谢大佬捏

**Reply by Gligameshes** [reply]
大佬，v1没有on_message模块该咋实现呜呜呜

**Reply by yanyongyu** [reply]
on_message是nb的。。

```python
from nonebot import on_message
from nonebot.adapters.onebot.v11 import GroupMessageEvent

def rule(event: GroupMessageEvent):
	if event.group_id in {...}:
		return True
	return False

matcher = on_message(rule)
```

**Reply by Gligameshes** [reply]
https://docs.nonebot.dev/api/
https://v2.nonebot.dev/docs/api/index
我是通过pip 装的nonebot 没有装nonebot2  也可以用on_message吗
提示ImportError: cannot import name 'on_message' from 'nonebot' 呜呜呜

**Reply by yanyongyu** [reply]
nb1的话，应该是`bot.on_message("group")`

**Reply by Gligameshes** [reply]
大佬，这个`bot.on_message("group")`是调用的onebot的api吗，为什么我再onebot的api文档中找不到on_message；还有`bot.on_message("group")这个只能监听所有群，要设置成监听单独某个群该怎么加参数，还是说只能在下面被修饰的函数里添加筛选群号的条件

**Reply by yanyongyu** [reply]
这个是aiocqhttp的方法，你可以在函数里自己过滤

**Reply by Gligameshes** [reply]
好的 ，谢谢
